sdk/go/Makefile

```diff
@@ -40,9 +43,12 @@ test_fast:: $(TEST_ALL_DEPS)
 test_auto:: $(TEST_ALL_DEPS)
 	@$(GO_TEST) $(TEST_AUTO_PKGS)
 
-dist::
-	go install -C pulumi-language-go \
-		-ldflags "-X github.com/pulumi/pulumi/sdk/v3/go/common/version.Version=${VERSION}" ${LANGHOST_PKG}
+dist:: ../../bin/pulumi-language-go
```

Comment:
I think this is going to break `make build_local` again.  That should keep working.

Reply:
It shouldn't change the behavior of `make build_local`. The implementation of `dist` has been changed but it's behavior should be identical.

I'm trying to test now.

Reply:
`make build_local` uses `dist` under the hood.  I think it will complain about `pulumi-language-go` being the same file when trying to `cp` it.